app/jobs/perilune/tasks/executor_job.rb

```diff
@@ -9,7 +11,11 @@ class UndefinedTask < StandardError; end
 
       def perform(task_id)
         @task_id = task_id
-        executor.execute
+
+        @duration = Benchmark.realtime do
+          executor.execute
+        end
```

Comment:
@duration is in `second` unit. Later in the code, we convert it manually in `ms`. Why?
- Trifle can store only `integer`. So if the task takes less than one second to run, we receive smth like `0.0014354` in duration. In ms, it does `1.4354` (and we round it) to have an int.